options/conversion.yaml

```
@@ -0,0 +1,2 @@
+Options:
+  - ReplaceIdentifiersWithUUIDs: false
```

```suggestion
ReplaceIdentifiersWithUUIDs: false
```
like suggested in the other PR something like this should be enough